hardhat.network.ts

```diff
@@ -1,5 +1,7 @@
 import { HardhatUserConfig } from 'hardhat/config';
 const mnemonic = process.env.HDWALLET_MNEMONIC;
+const infuraApiKey = process.env.INFURA_API_KEY;
```

Comment:
why was this re-introduced? not needed

Reply:
I'm not sure why it was was re-introduced? It was just present in my files.

Removed.